src/main/java/com/jcabi/github/RtContents.java

```diff
@@ -256,16 +256,21 @@ private Content content(
         final String path, final String ref
     ) throws IOException {
         final String name = "ref";
-        return new RtContent(
-            this.entry.uri().queryParam(name, ref).back(), this.owner,
-            this.request.method(Request.GET)
+        RtContent content = null;
+        final JsonStructure structure = this.request.method(Request.GET)
                 .uri().path(path).queryParam(name, ref).back()
                 .fetch()
                 .as(RestResponse.class)
                 .assertStatus(HttpURLConnection.HTTP_OK)
                 .as(JsonResponse.class)
-                .json().readObject().getString("path")
-        );
+                .json().read();
+        if (structure instanceof JsonObject) {
```

Comment:
@sadovnikov We try to avoid `instanceof`. Let's use something like:
```java
    if (ValueType.ARRAY == jsonStructure.getValueType()) {
            contentPath = path;
        } else if (ValueType.OBJECT == jsonStructure.getValueType()) {
            contentPath = ((JsonObject) jsonStructure).getString("path");
        }
```